rust/operator/src/container.rs

```diff
@@ -724,8 +930,15 @@ impl ContainerConfig {
                     format!(
                         "-javaagent:/stackable/jmx/jmx_prometheus_javaagent-0.16.1.jar={metrics_port}:/stackable/jmx/{role}.yaml",
                     )];
-                if let Some(Some(memory_limit)) =
-                    resources.limits.as_ref().map(|limits| limits.get("memory"))
+
+                if hdfs.has_kerberos_enabled() {
+                    jvm_args.push(
+                        "-Djava.security.krb5.conf=/stackable/kerberos/krb5.conf".to_string(),
```

Comment:
Do we want to factor this out, or respect `HADOOP_OPTS` somehow?

Reply:
I have to admit, I didn't fully get that.
Just for context, this is the reason we split HADOOP_OPTS and HDFS_NAMENODE_OPTS https://github.com/stackabletech/hdfs-operator/blob/06ca693c1205b2474fd24a3d4ab5a7a37b3d20cb/rust/operator/src/container.rs#L662-L695

Reply:
Yes, but.. this will break existing users who already rely on setting a `HADOOP_OPTS` override, in the sense that their settings will no longer take effect, right? Unless I'm missing something here.

Reply:
I a users currently overrides HADOOP_OPTS this override will still take precedence, so it should not break any existing setup. The implications is that some parts might not work because the miss the `java.security.krb5.conf` setting *when enabling kerberos*.
But i think this is the usual risk when using overrides.

Reply:
OK.